pkg/api/types.go

```diff
@@ -724,6 +726,10 @@ type SecretVolumeSource struct {
 	// Name of the secret in the pod's namespace to use.
 	// +optional
 	SecretName string
+	// same as above, just matches configmap naming for consistency
```

Comment:
I'll go ahead and remove this. The spec does say "name" should be used with a secret, so if "secretName" is not going to work then I suppose a new API object will be required.